### PR TITLE
Embedded msgs example

### PR DESCRIPTION
**Pull request by devcsomnicg**
## Description

- **Release Notes**
> We've integrated the Embedded view for card functionality into the system.

> Update the sample app

- **Test Steps**
> Start by launching the sample app, then select the 'Embedded Msgs' option, where you'll find the card view for embedded messaging.

### CODE REVIEW

**Comment by mprew97** [comment]
@devcsomnicg can we resolve the conflicts here please? Will review after the changes have been pulled back in.

**Review by mprew97 — CHANGES_REQUESTED** [review]
Are we trashing the `Card notification` and `Banner notification` PRs? Notice the card component code is in here but there's been no conflict resolution on those old PRs. Please close those out if we don't intend to merge them separately (which is fine by me at this point).

File: src/components/card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          <button
+            disabled={disableSecondaryBtn}
+            style={
+              disableSecondaryBtn
+                ? {
+                    ...defaultButtonStyles,
+                    ...secondaryBtnStyle,
+                    color: 'grey'
+                  }
+                : { ...defaultButtonStyles, ...secondaryBtnStyle }
```
[the remark this answers is not in the document]
Comment (devcsomnicg) [reply src/components/card/index.tsx:114]: Resolved.

File: src/embedded/embeddedManager.ts [hunk 2]
```diff
@@ -70,6 +70,12 @@ export class EmbeddedManager {
     this.actionListeners.push(actionHandler);
   }
 
+  // private notifyUpdateDelegates() {
```
[the remark this answers is not in the document]
Comment (devcsomnicg) [reply src/embedded/embeddedManager.ts:72]: Removed.

File: react-example/src/views/EmbeddedMsgs.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      <h1>Fetch Embedded Msgs</h1>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'flex-start'
+        }}
+      >
+        {messages.length &&
+          messages.map((message: any, index: number) => (
```
[the remark this answers is not in the document]
Comment (devcsomnicg) [reply react-example/src/views/EmbeddedMsgs.tsx:109]: Resolved.

File: react-example/src/views/EmbeddedMsgs.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+            <Card
+              key={index.toString()}
+              cardStyle={{ margin: 0 }}
+              text={message.messages[0]?.elements?.text[0]?.text}
```
[the remark this answers is not in the document]
Comment (devcsomnicg) [reply react-example/src/views/EmbeddedMsgs.tsx:113]: This will change once PR: https://github.com/Iterable/iterable-web-sdk/pull/243 will merge.

**Review by mprew97 — APPROVED** [review]

**Pull request merged**